datahub/search/exportcountryhistory/serializers.py

```diff
@@ -0,0 +1,41 @@
+from logging import getLogger
+
+from django.utils.translation import gettext_lazy
+from rest_framework import serializers
+
+from datahub.search.serializers import (
+    EntitySearchQuerySerializer,
+    SingleOrListField,
+    StringUUIDField,
+)
+from datahub.search.utils import SearchOrdering, SortDirection
+
+logger = getLogger(__name__)
+
+
+class SearchExportCountryHistorySerializer(EntitySearchQuerySerializer):
+    """Serializer used to validate ExportCountryHistory search POST bodies."""
+
+    default_error_messages = {
+        'no_empty_field': gettext_lazy(
+            'Request must include either country or company parameters',
+        ),
+    }
+
+    DEFAULT_ORDERING = SearchOrdering('history_date', SortDirection.desc)
+
+    SORT_BY_FIELDS = (
+        'history_date',
+    )
+
+    country = SingleOrListField(child=StringUUIDField(), required=False)
+    company = SingleOrListField(child=StringUUIDField(), required=False)
+
+    def validate(self, data):
+        """Serializer should have at least one parameter"""
+        if 'country' not in data and 'company' not in data:
```

Comment:
Could simplify this to something like:
```suggestion
        if not data.keys() & {'company', 'country'}:
```